kyo-combinators/shared/src/test/scala/kyo/FiberCombinatorTest.scala

```diff
@@ -19,7 +19,7 @@ class FiberCombinatorTest extends Test:
 
             "should construct from Future" in {
                 val future        = scala.concurrent.Future(100)
-                val effect        = Kyo.fromFuture(future)
+                val effect        = Kyo.fromFuture[Throwable, Int, Any](future)
```

Comment:
This is a good example of how the additional type param to allow effectful inputs introduces complexity.